.github/workflows/maven.yml

```diff
@@ -13,23 +13,23 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
-
     steps:
-    - name: Repository checkout
-      uses: actions/checkout@v2
-    - name: Cache local Maven repository
-      uses: actions/cache@v2
-      with:
-        path: ~/.m2/repository
-        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-        restore-keys: ${{ runner.os }}-maven-
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
-      with:
-        java-version: 1.8
-    - name: OWASP dependency checks
-      run: mvn dependency-check:check
-    - name: Build with Maven
-      run: mvn clean verify -Dgpg.skip
-    - name: Code coverage with Codecov  
-      uses: codecov/codecov-action@v1
+      - name: Repository checkout
+        uses: actions/checkout@v3
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-maven-
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
+          distribution: 'adopt'
+      - name: OWASP dependency checks
+        run: mvn dependency-check:check
+      - name: Build with Maven
+        run: mvn --batch-mode --update-snapshots package
```

Comment:
Is `mvn clean verify` preferred over `mvn --batch-mode --update-snapshots package`?